### DISCUSSION TITLE
How to disable default line wrap/break?

### DISCUSSION THREAD

**Discussion by seuristic** [NvChad/NvChad#2607] [Q&A] [answered]
I want to make the line extend to its maximum width not breaking down to the window's width. How can I achieve it?

Thanks!

**Comment by siduck** [comment]
@mshahanwaz how do u do it in neovim in general?

**Reply by seuristic** [reply]
Like `:set nowrap`?
I am asking how can I add it to `init.lua` file to work on nvim startup.

**Comment by seuristic** [comment, marked as answer]
Got it!
Added these lines in `custom/init.lua` file and now width is extending to it's full length
```lua
local opt = vim.opt
opt.wrap = false
``` 

Thanks.
